Dockerfile

```diff
@@ -72,27 +90,16 @@ RUN apt-get update && \
         ca-certificates \
         tzdata \
         libreadline8 \
-        tini && \
+        tini  \
+    && \
+    rm -rf /var/lib/apt/lists/* && \
```

Comment:
This directory is not actually cleared with `/etc/apt/apt.conf.d/docker-clean`, see : 

```
docker run -ti --rm debian:bullseye-slim cat /etc/apt/apt.conf.d/docker-clean
[...]
DPkg::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
APT::Update::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };

Dir::Cache::pkgcache "";
Dir::Cache::srcpkgcache "";
```